### ISSUE TITLE
Football Armor is bulletproof to pistols

### ISSUE THREAD

**Issue by Maleclypse** [CleverRaven/Cataclysm-DDA#54909]
### Describe the bug

21 Ballistic protection

### Steps to reproduce

Spawn football armor
Look at stats
Marvel at the ballistic protection

### Expected behavior

Not bullet proof.

### Screenshots

![image](https://user-images.githubusercontent.com/54345792/151675092-5bf42520-735e-4747-b99c-288baf6783a3.png)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: 761fb51 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Mythical Martial Arts [MMA],
    No Rail Stations [No_Rail_Stations],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Stats Through Skills [StatsThroughSkills]
]

**Comment by Maleclypse** [comment]
Western boots also seem to have 5 ballistic protection.

**Comment by Maleclypse** [comment]
Arming Pants 5 Ballistic
Backpack 6 Ballistic

**Comment by jackster483** [comment]
Realistically, what should the ballistic value be for all of these?

**Comment by Maleclypse** [comment]
probably half or so for all of them.

**Comment by catdach** [comment]
Ok so ,an item's damage protection values are calculated as the protective value of its `"material"` multiplied by the item's `"material_thickness"`. (I think if the item is made of multiple materials the material thickness is divided by the number of materials?) 
The way I see it, there are 3 options for changing the bullet protection values of an item:

1. Change the base bullet resist of the material (this will also effect **every other item that use this material**)
2. Change the material thickness of the item (this will also effect the protection from **every other type of damage**)
3. Change/add to/subtract from, the materials that the item is made of (see parentheses in option 2)

the football armor uses half "cotton" and half "thermoplastic resin"(which really just uses the proctection values of "Polymer resin") as materials, and it's material thickness is 6 (so `(3*cotton proc)+(3*resin proc)`). The bullet resist for resin is 6 which seems a little high to _me_ as someone with **no  f-ing clue** how bullet resistant plastic resin is. For reference, the bullet resist for  cotton is 1, most metals, leather, and wood are 2, steel and "Synthetic Fabric" are 3, Kevlar is 5, and ballistic glass is 9.

**Comment by catdach** [comment]
Where was I going with that? oh yeah, you have to pick which option you want for each individual item. Cant just change the individual protection values of individual items.

TLDR: Its basically make the material itself less protective against bullets, or make the item less protective overall.

**Comment by jackster483** [comment]
How did you figure out the formula for calculating the protection values? I assume it's not included in the JSON.

**Comment by catdach** [comment]
> How did you figure out the formula for calculating the protection values? I assume it's not included in the JSON.

Mostly intuition and deduction from comparing the numbers in [materials.json](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/materials.json), the [respective json files](https://github.com/CleverRaven/Cataclysm-DDA/tree/master/data/json/items/armor) for specific items, and the actual in-game values. There is also [a little bit of documentation](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/DEVELOPER_FAQ.md#how-armor-protection-is-calculated) I'm not sure how accurate the part about weighted averages is.(I'm not exactly sure that my formula for items with multiple materials is correct either, I would need to do a bit more extensive testing)

I certainly didn't go diving into the C++ or anything, though it might be a good idea if I want the real formula 😅
